test/Groups/conjugation.jl

```diff
@@ -33,19 +33,19 @@
   cc = conjugacy_class(G,x)
 
   @test length(cc) == 3
-  @test Set(elements(cc)) == Set([x^y for y in G])
+  @test Set(collect(cc)) == Set([x^y for y in G])
```

Comment:
Does this work, too?
```suggestion
  @test Set(collect(cc)) == Set(x^y for y in G)
```

Reply:
Maybe drop the `collect` here too?